src/arch/xtensa/include/arch/debug/panic.h

```diff
@@ -56,7 +56,7 @@ static inline void arch_dump_regs(void *dump_buf, uintptr_t stack_ptr,
 {
 	fill_core_dump(dump_buf, stack_ptr, epc1);
 
-	dcache_writeback_region(dump_buf, ARCH_OOPS_SIZE);
+	dcache_writeback_region((__force void __cache *)dump_buf, ARCH_OOPS_SIZE);
```

Comment:
why```__force void __cache``` when the type is already void *

Reply:
exactly because of that. `void *` doesn't have a `__cache` annotation so using it in place of a `void * __cache` argument causes a sparse warning. So, we need to cast this to `void * __cache`. And `__force` is also a sparse annotation to explicitly allow such cross-address-space type-casts.